client/src/components/NavBar/style.ts

```diff
@@ -0,0 +1,86 @@
+import { Classes, Colors } from "@blueprintjs/core";
+import { Chip } from "czifui";
+import styled, { css } from "styled-components";
+import { GRAY, PT_GRID_SIZE_PX, PT_TEXT_COLOR } from "./theme";
+import { HEADER_HEIGHT_PX } from "../../globals";
+
+export const Wrapper = styled.div`
+  background-color: ${PT_TEXT_COLOR};
+  height: ${HEADER_HEIGHT_PX}px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+`;
+
+export const MainWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  height: inherit; /* Take up full height of parent. */
+  justify-content: space-between;
+  padding: 0 16px;
+`;
+
+export const Left = styled.span`
+  align-items: center;
+  display: flex;
+  gap: 32px;
+
+  a {
+    display: flex; /* Ensures the anchor wrapping the logo has correct line height. */
+  }
+`;
+
+export const Right = styled.span`
+  align-items: center;
+  display: flex;
+  gap: 24px;
+`;
+
+export const Nav = styled.span`
+  display: flex;
+  gap: 16px;
+`;
+
+const button = css`
+  display: inline-block; /* Wrapper to mimic line height of children. */
+
+  .${Classes.BUTTON}.${Classes.MINIMAL} {
+    background: none;
+    border-radius: 0;
+    color: ${GRAY.D};
+    font-size: 13px;
```

Comment:
Adjusting font-size doesn't seem to do anything.

Reply:
You mean changing the font size doesn't make it match the data portal navbar?

Can you use chrome devtool to inspect the css diff between explorer and DP to see if there's any difference? Another possibility is that Explorer is not downloading the font file needed :thinking:

Reply:
I meant that changing the font size here isn't changing it in the UI - I tried decreasing it to make it match the data portal navbar.

Reply:
Ah got it!

I have a fix [PR here](https://github.com/chanzuckerberg/single-cell-explorer/pull/386) PTAL thank you!

Reply:
you da man!